src/main/java/com/crowdin/client/reports/model/GroupTaskUsageGenerateReportRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.crowdin.client.reports.model;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.Date;
+import java.util.List;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class GroupTaskUsageGenerateReportRequest extends GenerateReportRequest {
+    private String name = "group-task-usage"; // Note: The example shows "group-translation-costs-pe"
+    private Schema schema;
+
+    @Data
+    public static class Schema {
+        private List<Long> projectIds;
+        private Unit unit;
+        private Currency currency;
+        private ReportsFormat format;
+        private BaseRatesForm baseRates;
+        private List<IndividualRate> individualRates;
+        private NetRateSchemes netRateSchemes;
+        private Boolean excludeApprovalsForEditedTranslations;
+        private Boolean preTranslatedStringsCategorizationAdjustment;
+        private String groupBy;
+        private Date dateFrom;
+        private Date dateTo;
+        private List<Long> userIds;
+    }
```

Comment:
As far as I can see, this schema is different from what is shown in the documentation for this report type.

https://support.crowdin.com/developer/enterprise/api/v2/#tag/Reports/operation/api.groups.reports.post

<img width="866" height="1001" alt="Screenshot 2025-10-15 at 15 05 59" src="https://github.com/user-attachments/assets/a6188589-de20-4e66-a56a-0eb5d92a4c07" />